pkg/common/wrapper/ssh_wrapper.go

```diff
@@ -0,0 +1,57 @@
+package wrapper
+
+import (
+	"bytes"
+	"fmt"
+
+	"golang.org/x/crypto/ssh"
+)
+
+const (
+	// Port is default port to be used of ssh communication
+	Port = "22"
```

Comment:
`Port` and `Network` variables are used by out of this package?
If not, it seems they do not need to be global